### DISCUSSION TITLE
INFO Skipping configuration for protected branch...

### DISCUSSION THREAD

**Discussion by stefkiourk** [supabase/supabase#31251] [Questions]
I am changing the config.toml file to enable/disable different properties of auth and storage. However, I'm getting the message "INFO Skipping configuration for protected branch..." when I'm pushing the changes to main. I've tried creating another branch, which correctly picks up and reflects the changes, however when merging back to main, I get the same message. Example property: under [auth], enable_anonymous_sign_ins = true.

If "main" is my production branch, and I would like a change in the config, why would it skip it and consider it protected, even with normal PR merges? In case I am misunderstanding the process, how would one go about changing project configuration via Supabase branches?

Thank you.

**Comment by laudiacay** [comment]
following this- we're getting the same issue!

**Reply by Aptivisio** [reply]
Hey there! Checking if you had any luck fixing this. Having the same issue, hoping to avoid working through github actions..

**Comment by laudiacay** [comment]
We just moved off supabase to rds and alembic heh

**Comment by xanderdeseyn** [comment]
Also running into this. It's crazy how I can't find any other issues about this. This makes the whole branching feature useless? Code is no longer config.

**Reply by joristt** [reply]
[Docs are somewhat vague about this IMO](https://supabase.com/docs/guides/deployment/branching/configuration#remote-specific-configuration), but you have to specify the project id for your production/main branch explicitly in the config for this to work.

```
[remotes.main]
project_id = "your-project-ref"
```

This fixed it for me.
